### PR TITLE
#849 changed trash icon

### LINKED ISSUE #849: Remove Font Awesome Icons - ProposedSolutionView
### Description

We wanna remove font awesome icons as a dependency and replace them with icons from MUI. This ticket is to replace all instances of a font awesome icon in ProposedSolutionView.tsx with a corresponding version of that same icon from the library of MUI icons

Resources: https://mui.com/material-ui/material-icons/

### Acceptance Criteria

All uses of fontawesome icons are removed from ProposedSolutionView.tsx.
All of the icons have been replaced with MUI icons.
All old imports are removed.

### Proposed Solution

N/A

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by chloezhong3**
## Changes

The trash icon was changed to another version of the same icon from the library of MUI icons. 

## Screenshots
BEFORE:
// no trash icon
Screen Shot 2023-04-05 at 8 38 46 PM

AFTER:
// whole image
Screen Shot 2023-04-05 at 8 37 22 PM

// smaller image
Screen Shot 2023-04-05 at 8 37 36 PM

_If you made UI changes you must post a screenshot of the whole page for each change in 1) a normal sized window and 2) the smallest possible window_

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] All commits are tagged with the ticket number
- [ ] No linting errors / newline at end of file warnings
- [ ] All code follows repository-configured prettier formatting
- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Screenshots of UI changes (see Screenshots section)
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] No `yarn.lock` changes (unless dependencies have changed)
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #849

### CODE REVIEW

**Review by ryanhowe28 — CHANGES_REQUESTED** [review]
The screenshot that you submitted on this PR does not show the icon you changed/supposed to change. Make sure you are changing the correct icon and submit a picture of it. The icon you are changing is the trash icon for the proposed solution view. Add a proposed solution to a Change request and you will see it.

File: src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx [hunk 1]
```diff
@@ -39,7 +38,7 @@ const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
           }}
           sx={{ position: 'absolute', right: 75 }}
         >
-          <FontAwesomeIcon icon={faTrash} size="lg" data-testid={'deleteIcon'} />
+          <DeleteIcon data-testid="DeleteIcon" />
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx:41]: Don't know if you need the data testid. Take it out and see if the tests still pass, if they fail then put it back

Comment (winniechuma) [reply src/frontend/src/pages/ChangeRequestDetailPage/ProposedSolutionView.tsx]: Yea, you're right! we just removed it, tested it, and it works.☠️☠️☠️ - Chloe

**Review by anthonybernardi — APPROVED** [review]
nice work!

**Pull request merged**